Source/include/Ably/ARTLog.h

```diff
@@ -6,15 +6,16 @@ NS_ASSUME_NONNULL_BEGIN
 
 /// :nodoc:
 typedef NS_ENUM(NSUInteger, ARTLogLevel) {
-    ARTLogLevelVerbose,
-    ARTLogLevelDebug,
-    ARTLogLevelInfo,
-    ARTLogLevelWarn,
-    ARTLogLevelError,
-    ARTLogLevelNone
-};
+    ARTLogLevelVerbose NS_SWIFT_NAME(verbose),
```

Comment:
Same as above - isn't it how swift translates those names by default?

Reply:
https://github.com/ably/ably-cocoa/pull/1955/commits/41589f4819abc90695b4d3d5233046358fc17c27